tests/snuba/test_organization_discover.py

```diff
@@ -69,7 +69,22 @@ def test(self):
                 'projects': [self.project.id],
                 'fields': ['message', 'platform'],
                 'start': (datetime.now() - timedelta(seconds=10)).strftime('%Y-%m-%dT%H:%M:%S'),
-                'end': (datetime.now() + timedelta(seconds=10)).strftime('%Y-%m-%dT%H:%M:%S'),
+                'end': (datetime.now()).strftime('%Y-%m-%dT%H:%M:%S'),
+                'orderby': '-timestamp',
+            })
+
+        assert response.status_code == 200, response.content
+        assert len(response.data['data']) == 1
+        assert response.data['data'][0]['message'] == 'message!'
+        assert response.data['data'][0]['platform'] == 'python'
+
+    def test_relative_dates(self):
+        with self.feature('organizations:discover'):
+            url = reverse('sentry-api-0-organization-discover', args=[self.org.slug])
+            response = self.client.post(url, {
+                'projects': [self.project.id],
+                'fields': ['message', 'platform'],
+                'range': '1d',
```

Comment:
Should we have tests for invalid API usage as well? Invalid fields, out of bounds ranges, and requesting start/end & range at the same time?

Reply:
Added basic tests for requesting both and invalid ranges (not out of bounds since we don't handle that)